itunespy/__init__.py

```diff
@@ -105,7 +106,7 @@ def lookup(
     try:
         json = r.json()['results']
         result_count = r.json()['resultCount']
-    except KeyError:
+    except (JSONDecodeError, KeyError):
```

Comment:
Can we instead catch this separately and return a different error for this specific case? Since we're not sure whether there's been a connection error or some other kind of error (throttled, API unavailable, etc) I think it'd make it clearer. Either we can declare our own custom exception that conveys that or just throw some `RuntimeError` with a message explaining what happened :)